src/test/java/com/example/module3/AirportTest.java

```diff
@@ -33,7 +32,8 @@
             new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, ExperimentalTypes.VTOL, ClassificationLevel.TOP_SECRET)
     );
 
-    private static PassengerPlane planeWithMaxPassengerCapacity = new PassengerPlane("Boeing-747", 980, 16100, 70500, 242);
+    private static PassengerPlane planeWithMaxPassengerCapacity = new PassengerPlane("Boeing-747", 980,
```

Comment:
Why break the last three parameters, not only the last one?
How about to break lines in lines 31 and 32?